fastlane/swift/Fastlane.swift

```diff
@@ -1491,7 +1491,7 @@ func captureIosScreenshots(workspace: String? = nil,
                            appIdentifier: String? = nil,
                            addPhotos: [String]? = nil,
                            addVideos: [String]? = nil,
-                           htmlTemplate: String = "/Users/josh/Projects/fastlane/fastlane/snapshot/lib/snapshot/page.html.erb",
+                           htmlTemplate: String? = nil,
```

Comment:
I normally say to remove this file from the PR because its automatically created on next release of _fastlane_ but I think it makes sense to include this one here this time 😊